lib/ain-evm/src/block.rs

```diff
@@ -246,7 +246,25 @@ impl BlockService {
             Some(reward)
         };
 
+        // add another entry for baseFeePerGas
+        let next_block_base_fee = match self
+            .storage
+            .get_block_by_number(&(first_block + U256::one()))?
```

Comment:
can you test if the first_block == latest_block?

Reply:
resolved at https://github.com/DeFiCh/ain/pull/2390/commits/e8243fb9d8e2baacf2a72758083a40fb946a7d2c